lms/static/js/learner_analytics_dashboard/LearnerAnalyticsDashboard.jsx

```diff
@@ -49,9 +49,15 @@ function getStreakIcons(count) {
   ));
 }
 
+function getStreakEncouragement(count) {
+  const action = (count > 0) ? 'Maintain' : 'Start';
+
+  return `${action} your active streak by`;
+}
+
 function getStreakString(count) {
   const unit = (count ===1) ? 'week' : 'weeks';
```

Comment:
Does the missing space mean we have no linting for JSX in the platform, or that we didn't configure it for the learner_analytics_dashboard?  Either way, should we note it in the clean-up story: https://openedx.atlassian.net/browse/LEARNER-3854 ?